examples/jenkins/pipeline/pipelinetemplate.json

```diff
@@ -19,6 +19,9 @@
         "creationTimestamp": null,
         "labels": {
           "name": "sample-pipeline"
+        },
+        "annotations": {
+          "downstream.openshift.io/deployments": "[{\"name\": \"frontend\", \"namespace\": \"\"}]"
```

Comment:
> I don't love that it's deployment specific. What's the equivalent annotation for builds? 

downstream.openshift.io/builds

that was the intent of the generic "downstream.openshift.io" namespace, with the /foo being the downstream resource type if we decide to add more.

but i'd be fine with a generic annotation where the the resource type was part of the value.  @spadgett were just throwing something up for discussion.